numpy/lib/tests/test_io.py

```diff
@@ -217,6 +218,22 @@ def roundtrip(self, *args, **kwargs):
                 self.arr_reloaded.fid.close()
                 os.remove(self.arr_reloaded.fid.name)
 
+    def load_non_npy(self):
```

Comment:
```suggestion
    def test_load_non_npy(self):
```

Reply:
Oh, that would be why the test still passed.  Thank you for catching that.